src/Vision/VNUtils.cs

```diff
@@ -1,4 +1,4 @@
-//
+﻿//
```

Comment:
You're adding a byte order marker here - which should not happen, because our .editorconfig is configured otherwise. Which editor are you using?

Also, please undo this change :)

Reply:
hmm... I'm using VSCode!

Reply:
fixed!

Reply:
> VSCode!

Yep, looks like VSCode doesn't fully support .editorconfig: https://github.com/editorconfig/editorconfig-vscode/issues/35 which is blocked on https://github.com/microsoft/vscode/issues/824...